spec/classes/mod/jk_spec.rb

```diff
@@ -55,11 +59,74 @@
       { :logroot => '/var/log/apache2' }
     end
 
-    it_behaves_like 'minimal resources'
+    mod_dir = '/etc/apache2/mods-available'
+    let (:mod_dir) { mod_dir }
+
+    it_behaves_like 'minimal resources', mod_dir
     it {
       verify_contents(catalogue, 'jk.conf', ['<IfModule jk_module>', '</IfModule>'])
     }
 
   end
 
+  shm_log_paths = {
```

Comment:
rspec-puppet didn't like the change:
https://travis-ci.org/puppetlabs/puppetlabs-apache/jobs/268018580#L799
https://travis-ci.org/puppetlabs/puppetlabs-apache/jobs/268018581#L680
'`shm_log_paths` is not available on an example group (e.g. a `describe` or `context` block). It is only available from within individual examples (e.g. `it` blocks) or from constructs that run in the scope of an example (e.g. `before`, `let`, etc).'

What is the correct way to reference local variables declared in `let` statements? Current form is [this](https://github.com/puppetlabs/puppetlabs-apache/pull/1671/files#diff-b5f561614a193ce787d91557fc9ed417R98)